modules/assim.sequential/R/sda_plotting.R

```diff
@@ -453,46 +466,50 @@ post.analysis.ggplot.violin <- function(settings, t, obs.times, obs.mean, obs.co
         `colnames<-`(c('2.5%','97.5%','Variables'))%>%
         mutate(means=one.day.data$means%>%unlist,
                Type="Data",
-               Date=one.day.data$Date%>%as.POSIXct())
+               Date=one.day.data$Date%>%as.POSIXct(tz="EST"))
```

Comment:
Wanted to check on this -- PEcAn defaults to assuming all time is in UTC. Hard coding eastern US seems unlikely to be the correct assumption.